plutus-tx-plugin/src/PlutusTx/Compiler/Builtins.hs

```diff
@@ -282,6 +284,8 @@ defineBuiltinTerms = do
     defineBuiltinTerm 'Builtins.lessThanEqualsByteString $ mkBuiltin PLC.LessThanEqualsByteString
     defineBuiltinTerm 'Builtins.emptyByteString $ PIR.mkConstant () BS.empty
     defineBuiltinTerm 'Builtins.decodeUtf8 $ mkBuiltin PLC.DecodeUtf8
+    defineBuiltinTerm 'Builtins.verifyEcdsaSecp256k1Signature $ mkBuiltin PLC.VerifyEcdsaSecp256k1Signature
+    defineBuiltinTerm 'Builtins.verifySchnorrSecp256k1Signature $ mkBuiltin PLC.VerifyEcdsaSecp256k1Signature
```

Comment:
```suggestion
    defineBuiltinTerm 'Builtins.verifySchnorrSecp256k1Signature $ mkBuiltin PLC.VerifySchnorrSecp256k1Signature
```

Reply:
I've fixed that (as well as the attendant golden test), but the issue with the parser roundtripping still happens.